Image.js

```diff
@@ -1,46 +1,46 @@
 module.exports = {
-  list: {
+  providers: {
     // DummyImage
     DummyImage: {
-      getImgUrl: function (size) {
+      getImageUrl: function (size) {
         'use strict';
         return 'https://dummyimage.com/'+size+'/000/fff';
       }
     },
     // LoremPicsum
     LoremPicsum: {
-      getImgUrl: function (size) {
+      getImageUrl: function (size) {
         'use strict';
         var [width,height]=size.split('x');
         return 'https://picsum.photos/'+width+'/'+height+'/?random';
       }
     },
     // FakeImg
     FakeImg: {
-      getImgUrl: function (size) {
+      getImageUrl: function (size) {
         'use strict';
         var [width,height]=size.split('x');
         return 'https://fakeimg.pl/'+width+'x'+height+'/384f66/ecf0f1/?text=Spaceholder&font=lobster';
       }
     }
   },
-  prvd: 'random',
-  getPrvd: function () {
+  provider: 'random',
```

Comment:
```suggestion
  currentProvider: 'random',
```
Hard to distinguish `this.provider` and `this.providers`.